Makefile.am

```diff
@@ -740,10 +741,13 @@ EXTRA_DIST += \
 	man/glib-event-glue.c
 
 # ------------------------------------------------------------------------------
+rootlibexec_LTLIBRARIES += \
+	libshared.la
```

Comment:
2016-06-21 16:28 GMT+02:00 Felipe Sateler <notifications@github.com>:

> shouldn't this library name include the systemd version string? otherwise
> it might happen that during RPM/dpkg updates processes are started that
> load the wrong version of this library, which will break given that we
> won't maintain compat on this.
>
> This is a good idea. I will add the version info as ${version}:0:0 or so.
>
Couldn't we use:

-version-info $(LIBSYSTEMD_CURRENT):$(LIBSYSTEMD_REVISION):$(LIBSYSTEMD_AGE)

and add some make-install magic which removes the .so symlink
for "libsystemd-shared".

This would guarantee that we'll never have a version mismatch.


-- 
Why is it that all of the instruments seeking intelligent life in the
universe are pointed away from Earth?